app/_how-tos/get-started-with-metering-and-billing.md

```diff
@@ -0,0 +1,253 @@
+  consumers:
+    - username: kong-air
+      keyauth_credentials:
+        - key: air-key
+    - username: kong-travel
+      keyauth_credentials:
+        - key: travel-key
+{% endentity_examples %}
+<!--vale on-->
+
+## Enable authentication
+
+Authentication lets you identify a Consumer so that you can apply rate limiting.
+This example uses the [Key Authentication](/plugins/key-auth/) plugin, but you can use any [authentication plugin](/plugins/?category=authentication) that you prefer.
+
+Enable the plugin globally, which means it applies to all {{site.base_gateway}} Services and Routes:
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  plugins:
+    - name: key-auth
+      config:
+        key_names:
+          - apikey
+{% endentity_examples %}
+<!--vale on-->
+
+## Enable rate limiting
+
+Set rate limits for the consumers. Free consumer gets 1,000 requests for example-service and premium gets 5,000 requests for the same service/api.
+
+Enable the [Rate Limiting plugin](/plugins/rate-limiting/) for the Consumer. 
+In this example, the limit is 5 requests per minute and 1000 requests per hour.
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  plugins:
+    - name: rate-limiting
+      consumer: kong-air
+      config:
+        month: 1000
+    - name: rate-limiting
+      consumer: kong-travel
+      config:
+        month: 5000
+{% endentity_examples %}
+<!--vale on-->
+
+## Configure ACL plugin
+
+Configure the ACL plugin to only allow access to the service for customers who have signed up, so our two consumers:
+
+<!--vale off-->
+{% entity_examples %}
+entities:
+  consumers:
+    - username: kong-air
+      acls:
+      - group: kong-air
+      keyauth_credentials:
+      - key: air-key
+
+    - username: kong-travel
+      acls:
+      - group: kong-travel
+      keyauth_credentials:
+      - key: travel-key
+{% endentity_examples %}
+<!--vale on-->
+
+^ ChatGPT recommended this to fix my ACL error, tested and it seems to work.
+
+## Create a meter
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
+1. Enable **API Gateway Requests**.
+
+This will pull in request proxied by your API Gateway in {{site.konnect_short_name}} to Metering & Billing.
+
+## Create a feature
+
+Creating a feature that is linked to our service:
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
+1. In the Metering & Billing sidebar, click **Product Catalog**.
+1. Click **Create Feature**.
+1. In the **Name** field, enter `example-service`.
+1. From the **Meter** dropdown menu, select "kong_konnect_api_request". 
+1. Click **Add group by filter**.
+1. From the **Group by** dropdown menu, select "service_name".
+1. From the **Operator** dropdown menu, select "Equals".
+1. From the **Value** dropdown menu, select "example-service".
+1. Click **Save**. 
+
+## Create a plan
+
+Creating plans for our feature.
+
+1. In the {{site.konnect_short_name}} sidebar, click **Metering & Billing**.
```

Comment:
Can we use nested list to separate bigger tasks? 
If not, I think we should split this into three sections:

1. Create Plan
2. Add Rate Card
3. Publish Plan